src/url.ts

```diff
@@ -1,9 +1,10 @@
-import url from 'url';
+// tslint:disable-next-line no-require-imports
+const {parse} = require('url');
```

Comment:
since this is not used in `polaris` and only in `neutron`.
how do you feel about me removing this file all together and move it into `neutron`?